protocol/communities/manager.go

```diff
@@ -680,10 +680,27 @@ func (m *Manager) Joined() ([]*Community, error) {
 	return m.persistence.JoinedCommunities(&m.identity.PublicKey)
 }
 
+func (m *Manager) UpdateLastOpenedAt(communityID types.HexBytes) error {
+	return m.persistence.UpdateLastOpenedAt(communityID)
+}
+
 func (m *Manager) Spectated() ([]*Community, error) {
 	return m.persistence.SpectatedCommunities(&m.identity.PublicKey)
 }
 
+func (m *Manager) CommunityUpdateLastOpenedAt(communityID string) (*Community, error) {
+	community, err := m.GetByIDString(communityID)
```

Comment:
I believe we should also check if community is nil.

Reply:
No need to address it, `community==nil` check is not needed anymore: https://github.com/status-im/status-go/pull/4544.